Cargo.toml

```diff
@@ -75,16 +76,12 @@ branch = "feature/assume-init"
 
 [patch.crates-io.miniconf]
 git = "https://github.com/quartiq/miniconf.git"
-rev = "9c826f8"
+branch = "feature/client-integration"
```

Comment:
Ok for now, but do freeze it to a `rev`.

Reply:
Agreed - I'd like to release `miniconf` after the `std-embedded-nal` gets released, as I think we're at a good 0.1.0 point release state.